jekyll/_cci2/workflows.md

```diff
@@ -17,8 +17,9 @@ A workflow is a set of rules for defining a collection of jobs and their run ord
 - Fan-out to run multiple jobs in parallel for testing various versions  
 - Fan-in for deployment to separate platforms with automated triggers 
 - Branch-level job execution with artifact sharing
+- Manual job approval to control deployments to separate environments
```

Comment:
It looks like this list started as a list of verbs and turned into a list of nouns somewhere during the ambiguous "fan-out" and "fan-in". Can we make them line up again? If we make them all verbs we can change "if you need to meet any of the following requirements:" to simply "if you need to:".